tests/store/test_file_store.py

```diff
@@ -106,50 +105,50 @@ def test_valid_root(self):
         try:
             file_store._check_root_dir()
         except Exception as e:  # pylint: disable=broad-except
-            self.fail("test_valid_root raised exception '%s'" % e.message)
+            self.fail("test_valid_root raised exception '%s'" % e.value.message)
 
         # Test removing root
         second_file_store = FileStore(self.test_root)
         shutil.rmtree(self.test_root)
-        with self.assertRaises(Exception):
+        with pytest.raises(Exception):
             second_file_store._check_root_dir()
 
     def test_list_experiments(self):
         fs = FileStore(self.test_root)
         for exp in fs.list_experiments():
             exp_id = exp.experiment_id
-            self.assertTrue(exp_id in self.experiments)
-            self.assertEqual(exp.name, self.exp_data[exp_id]["name"])
-            self.assertEqual(exp.artifact_location, self.exp_data[exp_id]["artifact_location"])
+            assert exp_id in self.experiments
+            assert exp.name == self.exp_data[exp_id]["name"]
```

Comment:
I am not certain, but I was under the impression that assertEqual prints more useful debug info when the assertion fails than the standard assert statement. Is this false?

Reply:
Addressed below. It does when using unittest. However, pytest resolves this by making assert have the same semantics